examples/minimal-tanstack-start/app/ssr.tsx

```diff
@@ -4,7 +4,12 @@ import {
   defaultStreamHandler,
 } from "@tanstack/start/server";
 
+import { extractRouterConfig } from "uploadthing/server";
+
 import { createRouter } from "./router";
+import { uploadRouter } from "./server/uploadthing";
+
+globalThis.__UPLOADTHING = extractRouterConfig(uploadRouter);
```

Comment:
kinda hate this but it works... not sure where else we can import uploadRouter without leaking it to client bundle.